docs/ref/contrib/postgres/fields.txt

```diff
@@ -275,8 +275,8 @@ A more useful index is a ``GIN`` index, which you should create using a
 
 .. class:: HStoreField(**options)
 
-    A field for storing mappings of strings to strings. The Python data type
-    used is a ``dict``.
+    A field for storing mappings of strings to strings or nulls. The Python data type
+    used is a ``dict``. Database null values are represented as None in Python.
```

Comment:
It would be nice to find a phrasing that avoids the ambiguity of "mapping strings to strings" "or nulls" vs. "mapping strings" "to strings or nulls".